cvat/apps/engine/views.py

```diff
@@ -2363,7 +2343,12 @@ def _export_annotations(db_instance, rq_id, request, format_name, action, callba
                         db_storage = get_object_or_404(CloudStorageModel, pk=storage_id)
                         storage = db_storage_to_storage_instance(db_storage)
 
-                        export_to_cloud_storage(storage, file_path, filename)
+                        try:
+                            storage.upload_file(file_path, filename)
+                        except (ValidationError, PermissionDenied, NotFound) as ex:
+                            msg = str(ex) if not isinstance(ex, ValidationError) else \
+                                '\n'.join([str(d) for d in ex.detail])
```

Comment:
Because the validation error detail is a list with ErrorDetail. So when we convert it to a string, the result will be like `[ErrorDetail(string='some', code='invalid')]`. I guess the current implementation returns a more user-friendly message (e.g `some`). But I agree that there is small code duplication. I'll prepare another PR with an approach using a custom exception like CVATValidationError that inherits rest framework ValidationError and has re-implemented __str__ method.